assets/cubyz/tools/pickaxe.zig.zon

```diff
@@ -17,81 +17,68 @@
 	.parameters = .{
 		.{
 			.source = .density,
-			.destination = .swingTime,
+			.destination = .damage,
 			.matrix = .{
-				2.5, 2.0, 1.5, 1.0, 0x0,
-				2.0, 2.0, 1.5, 0x0, 0x0,
-				1.5, 1.5, 1.0, 0x0, 0x0,
-				1.0, 0x0, 0x0, 1.0, 0x0,
-				0x0, 0x0, 0x0, 0x0, 1.0,
+				2.0, 2.0, 1.5, 1.0, 0x0,
+				2.0, 1.5, 1.0, 0x0, 0x0,
+				1.5, 1.0, 0.6, 0x0, 0x0,
+				1.0, 0x0, 0x0, 0.4, 0x0,
+				0x0, 0x0, 0x0, 0x0, 0.2,
 			},
-			.factor = 0.25,
-			.method = .average,
+			.factor = 0.2,
+			.method = .sum,
 		},
 		.{
 			.source = .density,
 			.destination = .damage,
 			.matrix = .{
-				2.5, 2.0, 1.5, 1.0, 0x0,
+				2.0, 2.0, 1.5, 1.0, 0x0,
 				2.0, 1.5, 1.0, 0x0, 0x0,
-				1.5, 1.0, 0.5, 0x0, 0x0,
-				1.0, 0x0, 0x0, 0.0, 0x0,
-				0x0, 0x0, 0x0, 0x0, 0.0,
+				1.5, 1.0, 0.3, 0x0, 0x0,
+				1.0, 0x0, 0x0, 0.2, 0x0,
+				0x0, 0x0, 0x0, 0x0, 0.1,
 			},
-			.factor = 0.8,
+			.factor = 0.3,
 			.method = .average,
 		},
 		.{
 			.source = .hardness,
 			.destination = .maxDurability,
 			.matrix = .{
 				0.5, 1.0, 1.0, 1.0, 0x0,
-				1.0, 2.0, 1.5, 0x0, 0x0,
-				1.0, 1.5, 1.5, 0x0, 0x0,
+				1.0, 1.0, 1.0, 0x0, 0x0,
+				1.0, 1.0, 1.0, 0x0, 0x0,
 				1.0, 0x0, 0x0, 1.0, 0x0,
 				0x0, 0x0, 0x0, 0x0, 0.5,
 			},
-			.factor = 120,
-			.method = .average,
+			.factor = 15,
+			.method = .sum,
```

Comment:
This is not logical. Mixing low durability material with low durability material doesn't magically create even more durable material, unless this is a very specific mix of materials.